### DISCUSSION TITLE
Add number of discussions currently open (similar to issues/pull requests)

### DISCUSSION THREAD

**Discussion by Conza88** [community/community#3343] [Discussions]
![image](https://user-images.githubusercontent.com/51952482/107310825-4c95be80-6ae1-11eb-9eb8-95d3e40a36f2.png)

See above.

**Comment by PranamBhat** [comment]
Hi @Conza88 

Thank you for the feedback.

There's already a open discussion for the same. **GitHub** is looking into the same.

Thank you!
